From 8d54b1b2b33815ab952ad924b325207d0c1b1806 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Fri, 23 May 2025 23:26:19 +0100
Subject: [PATCH 1/1] data: Add a valid registered category to Waydroid.desktop
 for appstream

```
$ appstreamcli compose waydroid-install-prefix/
Automatically selected 'install/' as data output location.
WARNING: Metadata origin not set, using 'example'
Processing directory: install/
Composing metadata...
Run failed, some data was ignored.
Errors were raised during this compose run:
id.waydro.waydroid
  E: no-valid-category
Refer to the generated issue report data for details on the individual problems.
```

https://specifications.freedesktop.org/menu-spec/latest/category-registry.html

This fixes composing waydroid into a catalog when using appstream.
Without it waydroid will fail to be included in the catalog for distros
that use `appstream-generator`.
---
 data/Waydroid.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Waydroid.desktop b/data/Waydroid.desktop
index 6d7dd98..9174a0f 100644
--- a/data/Waydroid.desktop
+++ b/data/Waydroid.desktop
@@ -3,5 +3,5 @@ Type=Application
 Name=Waydroid
 Exec=waydroid first-launch
 Icon=waydroid
-Categories=X-WayDroid-App;
+Categories=X-WayDroid-App;Utility;
 X-Purism-FormFactor=Workstation;Mobile;
-- 
2.49.0

